pom.xml

```diff
@@ -16,7 +16,7 @@
   <properties>
     <revision>1.67</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.60.3</jenkins.version>
+    <jenkins.version>2.164.3</jenkins.version>
```

Comment:
I don't think the changes in this PR require this bump. What compilation errors were you getting previously?